### PR TITLE
CSCQVAIN-146: Qvain-Pilot: REM remove grey counter buttons

### PR DESCRIPTION
**Pull request by ShreyasDeshpande**
CSCQVAIN-146: Qvain-Pilot: REM remove grey counter buttons as user is getting confused what are they for. As stated in the story, not to remove the code but just comment it. so kept the commented code as it is

### CODE REVIEW

**Review by jppiiroinen — CHANGES_REQUESTED** [review]
The git keeps the history, so I would rather see the actual lines removed. by commenting the code we will have just code which is not in use.

File: src/widgets/Array.vue [hunk 1]
```diff
@@ -71,7 +71,7 @@
 						:key="'array-' + index" />
 				</b-list-group-item>
 				<div class="input__controls">
-					<b-btn class="input__control mr-2" type="button" variant="secondary"><font-awesome-icon icon="list" fixed-width/> <span>{{ minimum || "–" }} / {{ value.length }} / {{ maximum || "–" }}</span></b-btn>
+			<!--	<b-btn class="input__control mr-2" type="button" variant="secondary"><font-awesome-icon icon="list" fixed-width/> <span>{{ minimum || "–" }} / {{ value.length }} / {{ maximum || "–" }}</span></b-btn> -->
```
Comment (jppiiroinen) [inline src/widgets/Array.vue]: this is a test comment

**Review by jppiiroinen — APPROVED** [review]

**Comment by ShreyasDeshpande** [comment]
Removing commented code as the condition for keeping code commented is taken out from story description

**Comment by ShreyasDeshpande** [comment]
CSCQVAIN-146: Qvain-Pilot: REM remove grey counter buttons : Merging code as the code is verified

**Pull request merged**